<!DOCTYPE html>

<html lang="en">

<head>
	<title>Work Experience | Raymond Guo</title>

	<link rel="stylesheet" href="../styles/experience_style.css">
	<link rel="shortcut icon" href="../images/favicon.ico" type="image/icon">
	<link rel="icon" href="../images/favicon.ico" type="image/icon">

	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-122155105-1"></script>
	<script>
		window.dataLayer = window.dataLayer || [];
		function gtag(){dataLayer.push(arguments);}
		gtag('js', new Date());

		gtag('config', 'UA-122155105-1');
	</script>
</head>

<body>
	<div id = "wrapper">
		<div id = "navbar">
			<p id = "moving_menu_text">
				<a class="menu_item" id = "home_button" href="..">home</a> ·
				<a class="menu_item" id = "projects_button" href="../projects/">projects</a> ·
				<a class="menu_item" id = "experience_button" href="../experience/">work experience</a> ·
				<a class="menu_item" id = "academics_button" href="../academics/">academics</a> ·
				<a class="menu_item" id = "contact_button" href="../contact/">contact</a>
			</p>
		</div>
		<div class = "body">
			<div class = "animated fadeInDown">
				<p id = "top_menu_text">
					<a class="menu_item" id = "home_button" href="..">home</a> ·
					<a class="menu_item" id = "projects_button" href="../projects/">projects</a> ·
					<a class="menu_item" id = "experience_button" href="../experience/">work experience</a> ·
					<a class="menu_item" id = "academics_button" href="../academics/">academics</a> ·
					<a class="menu_item" id = "contact_button" href="../contact/">contact</a>
				</p>
				<img id = "experience_img" class = "img img-center" src="../images/experience.jpeg" width="150">
				<h1 class="heading">Work Experience</h1>
			</div>

			<div class = "animated fadeInUp">
				<h2 class="subheading">industry</h2>
				<div class = "all">
					<p class = "text">
						From May 2019 to August 2019, I will be a software engineering intern at
						<a class="link" href="https://www.golucid.co/" target="_">
							Lucid Software</a>, provider of the leading graphical web applications for visual productivity (<a class="link" href="https://www.lucidchart.com/pages/" target="_">LucidChart</a>)
							and design and brand management (<a class="link" href="https://www.lucidpress.com/" target="_">LucidPress</a>).
					</p>

					<p class = "text">
						Since September 2017, I have been a software developer for
						<a class="link" href="https://codebase.berkeley.edu/" target="_">
							Berkeley CodeBase</a>, a student organization at UC Berkeley that builds real-world
						computer science experience by providing software solutions for startups in the Bay area. This semester, I am a project manager, leading a team of developers to build a software product for a TBD company.
					</p>

					<p class = "text">
						Over the past few semesters, I have built a full-stack plasmid map visualization tool for
						<a class="link" href="https://www.zymergen.com/" target="_">
							Zymergen</a>, a machine-learning genomics research group, and a full-stack web administration portal for
						<a class="link" href="https://www.meetbeam.com" target="_">
							Beam</a>, a mobile banking startup.
					</p>

					<p class = "text">
						From May 2018 to August 2018, I was a network engineering intern at the
						<a class="link" href="https://www.jhuapl.edu/" target="_">
							Johns Hopkins University Applied Physics Laboratory (APL)</a>, a
						defense contractor specializing in space exploration and military technology research.
						I wrote a library of Tcl, Expect, and bash scripts to automate the management and configuration of hundreds of switches, firewalls, and access points,
						streamlining the process of setting up new devices and supervising existing devices in APL's network.
					</p>
				</div>
				<h2 class="subheading">research</h2>
				<div class = "all">
					<p class = "text">
						From January 2019 to May 2019, I was an undergraduate student researcher in the
						<a class="link" href="http://biomechanics.berkeley.edu/" target="_">
							Molecular Cell Biomechanics Laboratory
						</a> at UC Berkeley under <a class="link" href="http://bioeng.berkeley.edu/faculty/mohammad_mofrad" target="_">Prof. Mohammad Mofrad</a> and Dr. Mohammad Soheilypour.
						I worked on building a computational pipeline to assemble, annotate, and analyze paired-end DNA sequencing reads from the human gut microbiome metagenome. The data are pre-processed through this pipeline, then used to simulate gut metabolite compositions over time in Crohn's disease
patients to test potential cures and diets.
					</p>
					<p class = "text">
						From May 2016 to June 2017, I was a student researcher at the
						<a class="link" href="https://www.niddk.nih.gov/" target="_">
							National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK/NIH)
						</a> in the
						<a class="link" href="https://www.niddk.nih.gov/research-funding/at-niddk/labs-branches/genetics-development-disease-branch" target="_">
							Genetics of Development and Disease Branch</a>, under Dr. Jeffery Miller.
						I studied the effects of puromycin selection on the transcriptomes
						of primary human erythroblasts. To do this, we transduced cell lines with 2 different
						lentiviral vectors that conferred puromycin resistance and examined differences in RNA-Seq
						profiles by integrating bioinformatic scripts such as
						<a class="link" href="http://cole-trapnell-lab.github.io/cufflinks/" target="_">
							Cufflinks
						</a> and
						<a class="link" href="https://ccb.jhu.edu/software/tophat/index.shtml" target="_">
							TopHat</a>. I was awarded as a
						<a class="link" href="https://student.societyforscience.org/regeneronsts-scholars-2017" target="_">
							Regeneron Science Talent Search (STS) Semifinalist
						</a> for my work.
					</p>
					<p class = "text">
						From July 2015 to August 2015, I was a research intern at the
						<a class="link" href="https://geog.umd.edu/" target="_">
							University of Maryland Dept. of Geographical Sciences
						</a> under
						<a class="link" href="https://geog.umd.edu/facultyprofile/huang/chengquan" target="_">
							Dr. Chengquan Huang</a>. I studied Landsat data and images to determine the different types
							and amounts of land cover and vegetation/carbon dynamics throughout the United States.
					</p>
				</div>
				<h2 class="subheading">publications</h2>
				<div class = "publications">
					<ul class = "publication">
						<li>RL, Guo & Lee, Y. Terry & C, Byrnes & JL, Miller. (2017). Puromycin Selection Confounds the RNA-Seq Profiles of Primary Human Erythroblasts. Transcriptomics: Open Access. 05. 10.4172/2329-8936.1000140. [<a class="link" href="https://www.researchgate.net/publication/318656430_Puromycin_Selection_Confounds_the_RNA-Seq_Profiles_of_Primary_Human_Erythroblasts" target="_">link</a>]</li>
					</ul>
				</div>
				<div id = "infinity"> ~~~ ∞ ~~~ </div>
				<div id = "footer">	© 2019 Raymond Guo </div>
			</div>
		</div>
	</div>

</body>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<script src="../scripts/experience_script.js"></script>

</html>
